circuit-foundation/src/commonMain/kotlin/com/slack/circuit/foundation/NavigableCircuitContent.kt

```diff
@@ -54,25 +56,62 @@ public fun NavigableCircuitContent(
       unavailableRoute = unavailableRoute,
     )
 
-  if (backstack.size > 0) {
+  if (backstack.isEmpty) return
+
+  /*
```

Comment:
This whole doc is

![giphy](https://github.com/slackhq/circuit/assets/1361086/13a776d8-815e-4b5b-a379-df640576cf24)